### DISCUSSION TITLE
What is the use case for useTrail?

### DISCUSSION THREAD

**Discussion by iam-yan** [pmndrs/react-spring#1234] [Support] [answered]
In document `useTrail` is described as:
"Creates multiple springs with a single config, each spring will follow the previous one."

Yet it seems that custom delay, which describes the behavior of "follow", is not support.( i.e. delay: i => 500 * i )
To achieve this you have to chain `useTransition` or `useSpring`, like the use case in the following discussion:
https://github.com/pmndrs/react-spring/discussions/1083

Then what is `useTrail` for?

**Comment by j-o-o** [comment, marked as answer]
Correct me if I'm wrong, but it will take the props from the previous chain element. The other springs are "unique" to each other.

**Reply by drcmda** [reply]
exactly, a trail does not work with delays, but each spring, on each frame, takes the last interpolated value of the spring prior. since springs have friction it creates a natural chain that slows down with more items.

**Reply by iam-yan** [reply]
I c the point now. Thanks for the explanation!
